.github/workflows/ensure-triage-label.yml

```diff
@@ -0,0 +1,30 @@
+name: Ensure triage label is present
+on:
+  issues:
+    types:
+      - opened
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410 # v6
+        with:
+          script: |
+            const { data: labels } = await github.rest.issues.listLabelsOnIssue({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            });
+
+            console.log(JSON.stringify(labels, null, 2))
```

Comment:
🙂 Accidental debug log. We can keep it though.

Reply:
Removed